src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1278,7 +1278,7 @@ void Transformations::MainSnippets() {
 #if !defined(SNIPPETS_LIBXSMM_TPP) && defined(OPENVINO_ARCH_X86_64)
     const bool isMlpSeqSupported = is_infer_prc_supported_by_brgemm;
 #else
-    const bool isMlpSeqSupported = false;
+    const bool isMlpSeqSupported = any_of(config.inferencePrecision, ov::element::f32, ov::element::dynamic);
 #endif
```

Comment:
Can we add a separate branching for ARM64 platform? Because we still doesn't support MlpSeq on other platforms, and we must return `false` in such cases

Reply:
Done